doc/examples/jsdom/test/a11y.js

```diff
@@ -2,6 +2,8 @@
 const jsdom = require('jsdom');
 const { JSDOM } = jsdom;
 const assert = require('assert');
+const path = require('path');
+const axe = require(path.join(__dirname, '../../../../axe'));
```

Comment:
This is an example first of all. This needs to require axe-core. The fix you put in for our examples test script should make that possible.